tests/functional/spec/preview.spec.js

```diff
@@ -1,13 +1,27 @@
 import IntroductionPage from "../generated_pages/introduction_preview_hub/introduction.page";
+import IntroductionPageLinear from "../generated_pages/introduction/introduction.page";
 
 describe("Introduction preview questions", () => {
   const introductionSchema = "test_introduction_preview_hub.json";
+  const introductionSchemaLinear = "test_introduction.json";
   const showButton = 'button[data-ga-category="Preview Survey"]';
   const previewSummaryContent = "#summary-accordion-1-content";
-  beforeEach(() => {
-    browser.openQuestionnaire(introductionSchema);
-  });
+  const previewSectionTitle = ".ons-summary__group-title";
+  const previewQuestion = ".ons-summary__item";
 
+  it("Given I start a survey, When I view the preview page, Then all preview elements should be visible", () => {
```

Comment:
Are there not more?.

Reply:
We can add all possible to the first question.

Reply:
Don't have to be first question, just do multiple describes.

Reply:
I've added all possible fields on preview question now, except answer instruction check because it's not allowed on a Radio type answer, but since instruction won’t be used on live surveys do we need to check it?

Reply:
Answer instructions are used for Checkboxes.

Reply:
Added answer instruction check now.